CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.6.2
+
+- Load `fs-debug.js` capture script when `debug = true` option is used to match updated debug logging practices.
```

Comment:
Is there a KB article or some external doc that we can link to?

Reply:
Yeah, I linked it in the PR and such when I implemented this. I'll add it here for the sake of completeness.